gradle/dependencies.gradle

```diff
@@ -1,7 +1,6 @@
 ext {
 
   androidVersions = [
-      legacyMinSdkVersion     : 14,
```

Comment:
Any requirement for changing the legacy min SDK now?

Reply:
That's for the legacy `build` job was failing because now NN also declares `minSdkVersion` `19` so `app` test app was unable to build, which BTW is crashing at runtime I believe because latest changes introduced in https://github.com/mapbox/mapbox-navigation-android/pull/3040 We should discuss the plan for legacy deprecation, including the `app` module.

Reply:
Ah, here it is https://github.com/mapbox/mapbox-navigation-native/pull/1150#discussion_r431961756.

Since we're not planning any `0.x` releases, I think it's fine to stay this way.